modules/utils.sh

```diff
@@ -38,3 +38,13 @@ name_in_list() {
     done
     return 1
 }
+
+expand_var() {
+    local var="$1"
+
+    echo "${!var}"
+}
+
+uppercase() {
+    echo "$@" | tr '[:lower:]' '[:upper:]'
```

Comment:
Since we are using bash, this can be:
```
local parameters="$@"
echo ${parameters^^}
```
Or something along those lines. The trick is ${var^^}

I think we have just one caller for this, in:
./modules/service.sh:    service_upper=$(uppercase "$service")

That could be just "service_upper=${service^^} then.

Reply:
Yes, I actually fix this (along with other things that can be simplfied in bash) in a followup branch (since we switched to bash after I started this)